.circleci/docker/common/install_cudnn8.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# cuDNN license: https://developer.nvidia.com/cudnn/license_agreement
```

Comment:
This script seems to be hardcoded to work only in CUDA-11.5, can you please add check to throw an error if CUDA version is different than 11.5?

Reply:
By doing something like
`if [$1 -neq "11.5"];`

Reply:
This will be installed on cuda 11.3, 11.5 and 11.6 . Adding a check for CUDNN VERSION

Reply:
So what if someone will try to call this script to install CuDNN-8 for CUDA-10.2?